frontend/src/helpers/task/TaskPrefetcher.ts

```diff
@@ -0,0 +1,108 @@
+import { get_instance_list_from_task } from "../../services/instanceServices";
+import { getFollowingTask } from "../../services/tasksServices"
+
+export default class TaskPrefetcher {
```

Comment:
This class looks great Vitalii! really easy to read and understand how it works good job!